src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -207,6 +208,7 @@ private Conversion RewriteConversion(Conversion conversion)
             }
         }
 
+        [return: NotNullIfNotNull("type")]
```

Comment:
>[return: NotNullIfNotNull("type")] [](start = 8, length = 34)

why do we need this? The input and return values are already declared as not-nullable

Reply:
Reverting.

---
In reply to: [386583174](https://github.com/dotnet/roslyn/pull/42068#discussion_r386583174) [](ancestors = 386583174)